modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,379 @@
+--
++
+[,bash]
+----
+rpk topic alter-config <new-topic-name> --set redpanda.iceberg.mode=<topic-iceberg-mode>
+----
++
+[,bash,role=no-copy]
+----
+TOPIC            STATUS
+new-topic-name   OK
+----
+
+. Register a schema for the topic (optional). You must also set the `redpanda.iceberg.mode` topic property to `value_schema_id_prefix`.
++
+[,bash]
+----
+rpk registry schema create <subject-name> --schema </path-to-schema> --type <format>
+----
++
+[,bash,role=no-copy]
+----
+SUBJECT                VERSION   ID   TYPE
+new-topic-name-value   1         1    PROTOBUF
+----
++
+If you don't use a schema for the topic, select `key_value` for the topic Iceberg mode. Redpanda uses a simple schema for the Iceberg table, consisting of a column that stores the record’s metadata, key, and value.
+
+The Iceberg table name is the same as the Redpanda topic name. As you produce records to the topic, the data also becomes available in object storage for consumption by Iceberg-compatible clients. 
+
+== Schema support and mapping
+
+The `redpanda.iceberg.mode` property determines how Redpanda maps the topic data to the Iceberg table structure. You can either have the generated Iceberg table match the stucture of a Avro or Protobuf schema in the Schema Registry, or use the schemaless mode where Redpanda stores the record values as-is in the table. The JSON Schema format is not supported in this beta release. If your topic data is in JSON, it is recommended to use the `key_value` (schemaless) mode.
+
+=== Iceberg table modes
+
+For both `key_value` and `value_schema_id_prefix` modes, Redpanda writes to a `redpanda` table column that stores a single struct per record, containing nested columns of the metadata from each record, including the record timestamp, the partition it belongs to, and its offset. 
+
+In the `key_value` ("schemaless") mode, the `redpanda` metadata structs also contain both the record key and value. If you are associating a schema with the topic using the `value_schema_id_prefix` mode, the `redpanda` structs contain the record key only. Redpanda uses the matching schema to define table columns based on schema fields and then maps the record values to the corresponding columns.
+
+For example, if you produce to a topic according to the following Avro schema:
+
+```
+{
+    "type": "record",
+    "name": "ClickEvent",
+    "fields": [
+        {
+            "name": "user_id",
+            "type": "int"
+        },
+        {
+            "name": "event_type",
+            "type": "string"
+        },
+        {
+            "name": "ts",
+            "type": "string"
+        }
+    ]
+}
+
+```
+
+The `key_value` mode writes to the following table format:
+
+```
+CREATE TABLE ClickEvent (
+  redpanda OBJECT(
+    partition INT,
+    timestamp TIMESTAMP,
+    offset    LONG,
+    headers   ARRAY(OBJECT(KEY BINARY, VALUE BINARY NULLABLE)),
+    key       BINARY NULLABLE,
+    data      BINARY NULLABLE
+  )
+)
```

Comment:
This is no longer valid. `data` column is outside of `redpanda OBJECT`
```
CREATE TABLE ClickEvent (
  redpanda OBJECT(
    partition INT,
    timestamp TIMESTAMP,
    offset    LONG,
    headers   ARRAY(OBJECT(KEY BINARY, VALUE BINARY NULLABLE)),
    key       BINARY NULLABLE,
  ),
   data      BINARY NULLABLE
)
```

Reply:
It's also called `value` instead of `data` (so it makes sense with `redpanda.iceberg.mode=key_value` topic property)